utilities/ifix-migration-toolkit/README.md

```diff
@@ -0,0 +1,29 @@
+# National Dashboard Kafka Pipeline service
+
+National dashboard kafka pipeline will listen on records sent by national-dashboard-ingest service and sends them to their module specific topics to be indexed.
+
+### DB UML Diagram
+- NA
+
+### Service Dependencies
+- NA
+
+### Swagger API Contract
+
+- NA
+
+### Functionalities
+1. Listens on persist-national-records topic and redirects records to the module specific topic names to be indexed by kafka connector.
```

Comment:
Either write the correct information or keep the section blank. Remove the unrelated information

Reply:
minor change, to be taken up as part of code cleanup.